src/encoded/types/patient.py

```diff
@@ -200,6 +202,54 @@ def consent(self, request, consent):
     })
     def medications(self, request, medication):
         return paths_filtered_by_status(request, medication)
+    
+    @calculated_property(condition='medication', schema={
+        "title": "Medication duration",
+        "type": "array",
+        "items": {
+            "type": "string",
+        },
+    })
+    def medication_duration(self, request, medication):
+        
+        for object in medication:
+            medication_object = request.embed(object, '@@object')
+            date_format="%Y-%m-%d"
+            start_date=datetime.strptime(medication_object['start_date'],date_format )
+            end_date=datetime.strptime(medication_object['end_date'],date_format )
+            medication_duration=math.ceil((end_date-start_date).days/30)
+            duration_list=[] 
+            if 0<medication_duration<=3:
+                duration_list_item="1-3 month"
```

Comment:
In the issue, duration range starts at "0 - 3 month", not "1 - 3 month".
Do we have a definition on duration in month so we can implement code accordingly?
Also is it accurate to use math.ceil() function to modify the duration time before comparing?

Reply:
Update range to be 
0 <= duration < 3
3 <= duration < 6